person.rb

```diff
@@ -4,14 +4,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rentals
```

Comment:
- You don't need the parent_permission here. Could you remove this as this should not be a setter/getter method?